components/eam/src/physics/cam/micro_p3.F90

```diff
@@ -991,6 +991,11 @@ subroutine p3_main_part3(kts, kte, kbot, ktop, kdir, &
    real(rtype)    :: qm_incld     !in-cloud qm
    real(rtype)    :: bm_incld     !in-cloud bm
 
+   real(rtype), dimension(kts:kte) :: ze_rain, ze_ice
+
+   ze_rain = 1.e-22_rtype 
```

Comment:
Shouldn't these initial values be 10*log10(1e-22*1e18)=40 for consistency with the min value that's actually imposed when calculations are done?